app/views/questions/_questions_shadow.html.erb

```diff
@@ -0,0 +1,84 @@
+<% questions = questions || @popularquestions # accept local if present, default to instance %>
+<div id="notes" class="col-md-12">
+  <div class="row">
+
+      <div class="col-md-2">
+      </div>
+
+      <% questions.limit(2).each_with_index do |node,i| %>
+        <div class="card <% if @widget %>col-xs-3 col-sm-3 col-md-3 <% end %>col-lg-3 note note-container-question" style="margin-top:20px;">
+          <div class="card-body">
+
+          <div class="card-info">
+            <p class="meta"><%= render partial: "dashboard/question_node_meta", locals: { node: node } %></p>
+            <h5><a class="question" <% if @widget %>target="_blank"<% end %> href="<%= node.path(:question) %>"><%= node.title %></a></h5>
+            <div>
+            <div class="answer">
+              <% if params[:controller] == 'questions' %>
+                <a class="peopleResponse" href="<%= node.path(:question) %>#answers" title="Answers"><%= node.answers.length %> people responded</a>
+              <% end %>
+            </div>
+          </div>
+            <a class="btn btn-outline-secondary" href="/q/<%= node.id %>" >Read More</a>
+          </div>
+
+          </div>
+        </div>
+            
+      <% end %>
+
+      <div class="col-md-2">
+      </div>
+  </div>
+</div>
+
+<style>
+
+.card {
```

Comment:
Should we start to consolidate these styles and re-use them across cards? What do you think?

Reply:
Though the cards looks same but still there are many differences. The positioning and size of cards in different pages varies.

Reply:
OK, I still am thinking perhaps we should make a planning item for moving all the re-usable card layouts to one template folder so they're all next to one another. They are hard to find! Like `/app/views/items` or `/app/views/cards` maybe?

Reply:
But doesn't have to be in this PR

Reply:
Sure. Should I do this in next PR ?

Reply:
Or Le'ts wait a bit for all the cards and then shift them altogether to `/app/views/cards`